exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/metrics_exporter/__init__.py

```diff
@@ -107,7 +110,7 @@ def _get_temporality(instrument):
     return temporality
 
 
-def _get_type(value_type):
+def _get_type(value_type: Any):
```

Comment:
not 100% sure on the return type

```suggestion
def _get_type(value_type: Union[int, float]) -> MetricDescriptor.TypeValue:
```

Reply:
That would appear to be the correct return type according to [https://github.com/open-telemetry/opentelemetry-python/blob/1abdc02a92cd996f3924e1653896bb72123ac84e/opentelemetry-proto/src/opentelemetry/proto/metrics/v1/metrics_pb2.pyi#L141](https://github.com/open-telemetry/opentelemetry-python/blob/1abdc02a92cd996f3924e1653896bb72123ac84e/opentelemetry-proto/src/opentelemetry/proto/metrics/v1/metrics_pb2.pyi#L141), but for some reason the class variable is missing when imported (i.e. TypeValue does not exist as a property of MetricDescriptor). The same thing for TemporalityValue.

Reply:
Actually, this can be resolved with string literal annotations.